src/tests/functional/shared_test_classes/src/single_layer/dft.cpp

```diff
@@ -26,22 +26,20 @@ std::string DFTLayerTest::getTestCaseName(const testing::TestParamInfo<DFTParams
 }
 
 void DFTLayerTest::SetUp() {
-    InferenceEngine::SizeVector inputShapes;
+    InferenceEngine::SizeVector inputShape;
     InferenceEngine::Precision inputPrecision;
     std::vector<int64_t> axes;
     std::vector<int64_t> signalSize;
     ngraph::helpers::DFTOpType opType;
-    std::tie(inputShapes, inputPrecision, axes, signalSize, opType, targetDevice) = this->GetParam();
+    tie(inputShape, inputPrecision, axes, signalSize, opType, targetDevice) = GetParam();
```

Comment:
Now this code is reliy on ADL ...
What is `InferenceEngine::SizeVector` type will be changed ? It will potentially break search of tie function in `std` namespace ...

I personally prefer to use `std::tie`

Reply:
reverted those changes, I also don't like using without std